ReadMe.md

```diff
@@ -1,3 +1,116 @@
 # MAGE SDK for JavaScript (ES5)
 
-This is a work-in-progress.
+This is the official MAGE SDK for the web. It is to be used in conjunction with MAGE server
+applications. You can install [MAGE](https://www.npmjs.com/package/mage) from NPM.
+
+This SDK is completely written in ES5 to be compatible with the widest range of browsers.
+It uses the CommonJS style (`require()` calls and `module.exports`), and can thus be used with
+build systems like [Webpack](https://webpack.github.io/), which is the one we use and test on.
+
+## Compatibility
+
+This version of `mage-sdk-js` works with **MAGE >=1.1.0**.
+
+## Installation
+
+In the examples we use, we will assume your web code will live in the same project folder as
+your server application.
+
+```shell
+npm install --save mage-sdk-js
+```
+
+If you also want to install some MAGE module clients, you can install them via NPM as well:
```

Comment:
"MAGE client modules are used to connect to their MAGE server module counterparts"

To me, this order is much more natural than the one initially suggested here, but feedback welcome (cc @almirkadric)